supplemental-ui/.htaccess

```diff
@@ -92,6 +77,20 @@ RedirectMatch 301 ^/.?downloads/.*$ /download/index.html
 # Redirect plugin page to plugin portal
 Redirect 302 /plugins/index.html https://plugins.netbeans.apache.org/
 
+# former path to antora path after letting download to 
+Redirect 301 /about/ /front/main/about/
```

Comment:
Ah, I see you keep the redirections here. Why the change from the top of the file to the bottom of the file? Is that important (order or redirections is important in .htaccess files, AFAIR).